assets/app/view/game/round/auction.rb

```diff
@@ -152,9 +152,10 @@ def render_company_actions(company)
                       size: @current_entity.cash.to_s.size,
                     })
 
+          bid_str = @step.respond_to?(:bid_str) ? @step.bid_str(company) : 'Place Bid'
```

Comment:
shouldn't this be inside the if statement?

Reply:
I copied this line from line 142 from the same file as I needed this same functionality for bidding.  I've also tested and this is working correctly for my game and it didn't change anything for the games I tested (nor would I expect it to since none of them will implement #bid_str right now).

Reply:
it's only used in the context of the if statement on line 157, you should put it in there